cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -57,6 +57,7 @@ type options struct {
 
 	configSearchPaths  cli.StringSlice
 	librarySearchPaths cli.StringSlice
+	skipHook           cli.StringSlice
```

Comment:
This should be:
```suggestion
	skipHooks           cli.StringSlice
```
Since it contains 0 or more entries.